tensorflow/lite/delegates/webnn/BUILD

```diff
@@ -651,6 +651,7 @@ cc_test(
     ],
 )
 
+#Cannot test now since it has int32 input tensor
```

Comment:
Maybe we can make it more clear like: "Disable this test temporarily as it contains int32 input tensor, which is not supported in WebNN at present."